cmd/tools/grafana/grafana.go

```diff
@@ -399,13 +415,16 @@ func initImportVars() {
 	m := make(map[string]*Folder)
 
 	// default behaviour
-	if opts.dir == "grafana/dashboards" && opts.serverfolder.name == "" {
+	switch {
+	case opts.dir == "grafana/dashboards" && opts.serverfolder.name == "":
 		m[filepath.Join(opts.dir, "cmode")] = &Folder{name: "Harvest-" + harvestRelease + "-cDOT"}
 		m[filepath.Join(opts.dir, "cmode", "details")] = &Folder{name: "Harvest-" + harvestRelease + "-cDOT Details"}
 		m[filepath.Join(opts.dir, "7mode")] = &Folder{name: "Harvest-" + harvestRelease + "-7mode"}
 		m[filepath.Join(opts.dir, "storagegrid")] = &Folder{name: "Harvest-" + harvestRelease + "-StorageGrid"}
-	} else if opts.dir != "" && opts.serverfolder.name != "" {
+	case opts.dir != "" && opts.serverfolder.name != "":
 		m[opts.dir] = &Folder{name: opts.serverfolder.name}
+	case opts.dir != "" && opts.customizeDir != "":
+		m[opts.dir] = &Folder{name: opts.dir}
```

Comment:
`&Folder{name: opts.dir}` might be `&Folder{name: opts.customizeDir}` ?

Reply:
The folder's name is not used by the customize code since it uses the existing dashboard's sub-directory name